test/Microsoft.AspNet.Mvc.Core.Test/ControllerActionInvokerTest.cs

```diff
@@ -1958,18 +1959,37 @@ public void CreateActionResult_ReturnsObjectContentResult(Type type, object inpu
             Assert.Same(input, contentResult.Value);
         }
 
+        [Fact]
+        public async Task MaxAllowedErrorsIsSet_BeforeCallingAuthorizationFilter()
+        {
+            // Arrange
+            var expected = 147;
+            var filter = new MockAuthorizationFilter(expected);
+            var invoker = CreateInvoker(
+                filter,
+                actionThrows: false,
+                tempData: null,
+                maxAllowedErrorsInModelState: expected);
+
+            // Act & Assert
+            // The authorization filter asserts if MaxAllowedErrors was set to the right value.
```

Comment:
nit: could have used a mock with a callback to assert. That way the assert would have been in the test case and would be clear. Also you would not need to be a specific filter (authorization filter), it could have been a simple IFilter.